### PR TITLE
Feature/form

### PR DESCRIPTION
**Pull request by douglasvinicio**

### CODE REVIEW

File: components/RadioButton/RadioButton.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const RadioButton = (props) => {
+    const { changed, id, isSelected, label, value } = props
+    return (
+        <div className='RadioButton'>
+            <input id={id} onChange={changed} value={value} type='radio' checked={isSelected} />
+            <label style={{ margin: '0 5px' }} htmlFor={id}>
```
Comment (marceloglacial) [inline components/RadioButton/RadioButton.jsx:6]: Please use CSS instead of inline styles

File: package.json [hunk 2]
```diff
@@ -14,8 +14,10 @@
         "@next/font": "13.1.6",
         "eslint-config-next": "13.1.6",
         "next": "13.1.6",
+        "prettier": "^2.8.4",
         "react": "18.2.0",
-        "react-dom": "18.2.0"
+        "react-dom": "18.2.0",
+        "react-modal": "^3.16.1"
```
Comment (marceloglacial) [inline package.json:20]: Any additional dependency should be discussed with the team. Please create a modal component :)

File: styles/globals.css [hunk 3]
```diff
@@ -345,6 +346,23 @@ button.outline:disabled {
     justify-content: flex-start;
 }
 
+/* Solution for select list from Form.jsx 'Immigration Status' without the use of Javascript
```
Comment (marceloglacial) [inline styles/globals.css:349]: Please create a separated file for Forms

File: components/Form/Form.jsx [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+    function closeSuccessSubModal() {
+        setisSuccesfullSubmissionModalOpen(false)
+    }
+
+    const radioButtonHandler = (e) => {
+        setResideInCanada(e.target.value)
+    }
+
+    return (
+        <>
```
Comment (marceloglacial) [inline components/Form/Form.jsx:47]: The following div is already a wrapper, fragment no needed

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: components/RadioButton/RadioButton.jsx (hunk 1 above)
Comment (douglasvinicio) [reply components/RadioButton/RadioButton.jsx]: Done

File: components/Form/Form.jsx (hunk 4 above)
Comment (douglasvinicio) [reply components/Form/Form.jsx]: Done

File: components/RadioButton/RadioButton.jsx (hunk 1 above)
Comment (douglasvinicio) [reply components/RadioButton/RadioButton.jsx]: RadioButton.module.css created inside styles folder

File: package.json (hunk 2 above)
Comment (douglasvinicio) [reply package.json:20]: Modal created inside Form component and react-modal removed from project dependencies

**Review by marceloglacial — APPROVED** [review]

**Pull request merged**